js/reference.js

```diff
@@ -16,6 +16,17 @@
 
 var ERMrest = (function(module) {
 
+    module._contexts = Object.freeze({
```

Comment:
@RFSH I wonder if we should we apply Object.freeze to module._annotations as well? Annotations shouldn't be mutated either right?